types/compression-next/index.d.ts

```diff
@@ -0,0 +1,174 @@
+// Type definitions for compression-next 1.0
+// Project: https://github.com/expressjs/compression
+// Definitions by: Santi Albo <https://github.com/santialbo>
+//                 Rob van der Burgt <https://github.com/rburgt>
+//                 Neil Bryson Cargamento <https://github.com/neilbryson>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 Akira Miyakoda <https://github.com/AkiraMiyakoda>
```

Comment:
```suggestion
// Definitions by: Akira Miyakoda <https://github.com/AkiraMiyakoda>
```
just add a reference to original DT package owners, but you'll be the owner.